vroom/json.vroom

```diff
@@ -0,0 +1,120 @@
+Maktaba provides some simple functions to parse and format Vim values to JSON
+texts. These can parse and format numbers, strings, lists and dictionaries to
+their JSON equivalents.
+
+Currently, these functions depend upon access to Python, either via a version
+of Vim compiled with +python (or +python3), or by shelling out to the Python
+binary.
+
+As always, we start by installing Maktaba.
+
+  :set nocompatible
+  :let g:maktabadir = fnamemodify($VROOMFILE, ':p:h:h')
+  :let g:bootstrapfile = g:maktabadir . '/bootstrap.vim'
+  :execute 'source' g:bootstrapfile
+
+
+
+We can start by formatting a variety of Vim values as JSON. Any Vim value other
+than a Funcref can be formatted as a JSON text.
+
+  :function CheckEqual(a, b) abort
+  :  return maktaba#ensure#IsEqual(a:b, a:a)
+  :endfunction
+
+  :call CheckEqual(maktaba#json#Format('Hello, world'), '"Hello, world"')
+  :call CheckEqual(maktaba#json#Format(42), '42')
+  :call CheckEqual(maktaba#json#Format(42.0), '42.0')
+  :call CheckEqual(maktaba#json#Format([1, 2, 'foo']), '[1, 2, "foo"]')
+  :let g:json = maktaba#json#Format({'foo': {'bar': 'baz'}})
+  :call CheckEqual('{"foo": {"bar": "baz"}}', g:json)
```

Comment:
Fixed. (Also named them "actual" and "expected" in CheckEqual so it's slightly more obvious why I flipped the order.)